src/prisma/schema.prisma

```diff
@@ -1323,6 +1323,7 @@ model ProposalEvaluation {
   permissions          ProposalEvaluationPermission[]
   proposalNotification ProposalNotification[]
   actionLabels         Json?
+  failReasons          String[]                            @default([])
```

Comment:
Is this a list for future-proofing? Seems unecessary but is OK.

we know that we want multiple reviewers to give reasons, and we should be able to tie them back to the reviewer, so the chosen fail reasons should be stored someplace else